stdlib/result.gr

```diff
@@ -0,0 +1,63 @@
+export let isOk = (v) => {
+    match (v) {
+        | Ok(_)  => true
+        | _      => false
+    }
+};
+
+export let isErr = (v) => !isOk(v);
+
+export let toOption = (v) => {
+    match (v) {
+        | Ok(x)  => Some(x)
+        | _      => None
+    }
+}
+
+export let mapOk = (v, fn) => {
+    match (v) {
+        | Ok(x)  => Ok(fn(x))
+        | Err(e) => Err(e)
+    }
+}
+
+export let mapErr = (v, fn) => {
+    match (v) {
+        | Err(e) => Err(fn(v))
```

Comment:
Should just the `e` be passed to `fn`? Maybe that's a typechecker error?

Reply:
Good catch, there is a type checker error right now.